lib/typescript/tsconfig.json

```diff
@@ -1,5 +1,6 @@
 {
   "compilerOptions": {
-    "isolatedModules": true
+    "isolatedModules": true,
+    "lib": ["DOM", "ES5", "ES2015", "ES2016", "ES2017", "ES2018", "ESNext"],
```

Comment:
ES5 was released in 2009. Surely we don't need to target that any more?

Reply:
Can this be cleaned up with only recently released targets (say last three years)?